src/Sylius/Component/Channel/composer.json

```diff
@@ -22,7 +22,7 @@
     "require": {
         "php": ">=5.5.9",
 
-        "doctrine/orm": "2.4.*",
+        "doctrine/orm": "^2.4.8",
```

Comment:
Looks like ORM's exceptions are used in PhpDocs in `ChannelRepositoryInterface`.

`doctrine/common` is used too in `ChannelsAwareInterface` only.